typescript/prompt-iteration-promptfoo/src/wildlifeAssistant.ts

```diff
@@ -2,9 +2,9 @@ import OpenAI from "openai";
 import { ChatCompletionMessageParam } from "openai/resources";
 import "dotenv/config";
 
-import { Prompt } from "@literalai/client";
+import { LiteralClient, Prompt } from "@literalai/client";
 
-import { client } from "./index";
+export const client = new LiteralClient();
```

Comment:
I kept hitting an error because `client` was undefined and therefore could not read `client.instrumentation`. 
It tried to await imports, but it wouldn't let me (maybe it doesn't make sense). 

When I instantiated the client here, it worked.